### DISCUSSION TITLE
Github-Hosted Runners Throttled?

### DISCUSSION THREAD

**Discussion by bhjelstrom** [community/community#45542] [Actions]
### Select Topic Area

Product Feedback

### Body

We have been on a mission to move all of our Linux Image generation to GitHub runners. First we needed more cores, then more memory, then SSDs. We have all of that now, but now we seem to get throttled on every build. Our runners clone a large number of repos to build linux images from scratch. 

Are github-hosted runners throttled after a certain transfer threshold is crossed?

**Comment by jge162** [comment]
there may be limits on the amount of data that can be transferred per minute or per hour for GitHub-hosted runners. 

To verify if this is the case, you can try to contact GitHub support for more information on their network and resource usage limits.

[Runner Documentation](https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#usage-limits)

**Reply by jge162** [reply]
Below is a bot I believe @bhjelstrom   the link is to insulin. And she just copied what I wrote. Joined GitHub 25 min ago. FYI

**Comment by Elizabeth0780** [comment]
GitHub Actions usage is free for standard GitHub-hosted runners in public repositories, and for self-hosted runners. For private repositories, each GitHub account receives a certain amount of free minutes and storage for use with GitHub-hosted runners, [depending](https://insulin-store.com/) on the product used with the account.
